meson.build

```diff
@@ -208,7 +208,7 @@ dep_webp        = dependency('libwebp',        version : '>=0.5',   required : t
 dep_webpdecoder = dependency('libwebpdecoder', version : '>=0.5',   required : false)
 dep_zlib        = dependency('zlib',                                required : true)
 dep_zstd        = dependency('libzstd',        version : '>=1.4.0', fallback : ['libzstd', 'libzstd_dep'])
-dep_zip         = dependency('libzip',         version : '>=1.5.0', required : opt_vfs_zip, allow_fallback : true)
+dep_zip         = dependency('libzip',         version : '>=1.5.0', required : opt_vfs_zip, fallback : ['libzip', 'libzip_dep'])
```

Comment:
Can you explain what was the problem here and/or do you have a failed build log?

I think the problem with zstd was related to `meson.override_dependency()` misbehaving when the dependency is needed for both native and cross targets, but I don't remember the details. I wouldn't think it's the same problem here; IIRC libzip is only used by taisei itself.

Reply:
The relevant error line in the build log was: 
```
WARNING: Subproject 'libzip' did not override 'libzip' dependency and no variable name specified
```
I eventually traced back the error to an [issue you posted on the meson repo](https://github.com/mesonbuild/meson/issues/11189) which led me to believe this was the same problem.

Reply:
Was that built in a pristine container or just locally? In the later case you might have an older version of `libzip` checked out under `subprojects`, which doesn't call `meson.override_dependency()`. See if it still happens after a `meson subprojects update` or just `rm -rf subprojects/libzip`.

But if this happens under a totally clean checkout, that is very odd and needs further investigation.

Reply:
It does not happen after a clean checkout, so I removed the change from the PR